src/services/ui/src/api/submissionService.ts

```diff
@@ -75,7 +74,10 @@ const buildSubmissionPayload = <T extends Record<string, unknown>>(
         origin: "micro",
         ...data,
         ...userDetails,
-        proposedEffectiveDate: (data.proposedEffectiveDate as Date).getTime(),
+        submissionDate: seaToolFriendlyTimestamp(),
+        proposedEffectiveDate: offsetForUtc(
```

Comment:
I think my last comment may help this if I'm processing the logic right in my brain

Reply:
Yep, that was the idea I was going for! Wasn't aware that middle step of the else would break it so I bundled the default up in the signature, but this looks good